packages/slate-plugins/src/elements/mention/transforms/wrapMentionBrackets.ts

```diff
@@ -0,0 +1,63 @@
+import { Point, Range, Editor, Transforms } from "slate";
+
+const pointToStr = (point: Point | undefined) => {
+  if (!point) {
+    return "-";
+  }
+  let pointStr = "";
+  pointStr += point.path.join(".");
+  pointStr += " " + point.offset;
+  return pointStr;
+};
+
+const rangeToStr = (range: Range) => {
+  let rangeStr = "";
+  rangeStr += pointToStr(range.anchor);
+  rangeStr += " -> ";
+  rangeStr += pointToStr(range.focus);
+  return rangeStr;
+};
```

Comment:
Are these only used for console logging that will be removed? Perhaps a good idea to remove in that case.

Reply:
see comment above re. console logging

Reply:
will change all the console logs to console.debugs